model/transform/src/main/scala/aqua/model/transform/pre/ArgsProvider.scala

```diff
@@ -18,7 +18,7 @@ object ArgsProvider {
     // Variable name to store the value of the argument
     varName: String,
     // Type of the argument
-    t: DataType
+    t: DataType | StreamType
```

Comment:
not sure, but maybe we should have `ArgType` for args that unites DataType and StreamType

Reply:
I don't think it is beneficial. It is used just one time and this notation says that we can get data or stream from args pretty straightforward.